src/MUITheme/GymTheme.tsx

```diff
@@ -4,37 +4,38 @@ import variables from '../styles/base/export.module.scss';
 const { textColor, primaryColor, secondaryColor, accentColor, fontFamilyPrimary } = variables;
 
 declare module '@mui/material/styles' {
-	interface Palette {
-		accent: {
-			main: string;
-		};
-	}
+  interface Palette {
+    accent: {
+      main: string;
+    };
+  }
 
-	interface PaletteOptions {
-		accent?: {
-			main?: string;
-		};
-	}
+  interface PaletteOptions {
+    accent?: {
+      main?: string;
+    };
+  }
 }
 
 const GymTheme = createTheme({
-	palette: {
-		text: {
-			primary: textColor,
-		},
-		primary: {
-			main: primaryColor,
-		},
-		secondary: {
-			main: secondaryColor,
-		},
-		accent: {
-			main: accentColor,
-		},
-	},
-	typography: {
-		fontFamily: fontFamilyPrimary,
-	},
+  palette: {
+    mode: 'dark',
```

Comment:
Added here mode: 'dark', everything else is linted

Reply:
What do we need this mode for?

Reply:
It's because MUI by default makes inputs in the form dark and unreadable with our dark theme:
![image](https://github.com/user-attachments/assets/57e32770-b380-4591-84a7-831c18ef3259)
![image](https://github.com/user-attachments/assets/fe4fd30f-79d9-4d27-bddc-c5fefe1983c7)

Reply:
it would be better to add Input colors styling to the theme

Reply:
So, I figure out how to do it, but the cost is too much new code (the question is do we need it?):
in stead of:
```
mode: 'dark',
```
we would have:
```
components: {
    MuiFilledInput: {
      styleOverrides: {
        root: {
          backgroundColor: '#424242', // Set background color for input fields in dark mode
          '&:hover': {
            backgroundColor: '#616161', // Background on hover
          },
          '&.Mui-focused': {
            backgroundColor: '#333', // Background when focused
          },
        },
        input: {
          color: '#ffffff', // Text color inside input
        },
      },
    },
    MuiInputLabel: {
      styleOverrides: {
        root: {
          color: '#bdbdbd', // Label color in dark mode
          '&.Mui-focused': {
            color: '#ffffff', // Label color when focused
          },
        },
      },
    },
    MuiFormHelperText: {
      styleOverrides: {
        root: {
          color: '#bdbdbd', // Helper text color in dark mode
        },
      },
    },
  },
```